src/Transports/MassTransit.AmazonSqsTransport/Pipeline/AmazonSqsMessageReceiver.cs

```diff
@@ -133,7 +137,11 @@ async Task<int> ReceiveMessages(int messageLimit, ChannelExecutor executor)
         {
             try
             {
-                IList<Message> messages = await _client.ReceiveMessages(_receiveSettings.EntityName, messageLimit, _receiveSettings.WaitTimeSeconds, Stopping)
+                IList<Message> messages = await _client.ReceiveMessages(

```

Comment:
this isn't how we format code in MassTransit !!! :)

Reply:
LOL, I though you'd say it